sheet/Skill.qml

```diff
@@ -18,5 +21,22 @@ Attribute {
             amount: ranks
         }
     ]
+
+    Bonus {
+        id: rankBonus
+        name: root.name
+        amount: -ranks
```

Comment:
What do you mean?

On Mon, Jun 8, 2015 at 7:33 PM, jarjee <notifications@github.com> wrote:

> In sheet/Skill.qml
> <https://github.com/shocklateboy92/psychic-bear/pull/19#discussion_r31898667>
> :
>
> > @@ -18,5 +21,22 @@ Attribute {
> >              amount: ranks
> >          }
> >      ]
> > +
> > +    Bonus {
> > +        id: rankBonus
> > +        name: root.name
> > +        amount: -ranks
>
> I thought we wanted to avoid -ve amounts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/shocklateboy92/psychic-bear/pull/19/files#r31898667>.
>